corehq/apps/translations/app_translations/utils.py

```diff
@@ -161,11 +161,20 @@ def is_single_sheet(identifier):
     return identifier == SINGLE_SHEET_NAME
 
 
-def get_menu_or_form(app, identifier):
-    if '_' in identifier:
-        return get_form_from_sheet_name(app, identifier)
+def get_menu_or_form_by_sheet_name(app, sheet_name):
+    if '_' in sheet_name:
+        return get_form_from_sheet_name(app, sheet_name)
     else:
-        return get_module_from_sheet_name(app, identifier)
+        return get_module_from_sheet_name(app, sheet_name)
+
+
+def get_menu_or_form_by_unique_id(app, unique_id, row_type):
```

Comment:
Why not always pass sheet name, since that's always available, instead of having row_type be either Module/Form or sheet name, which is a little overloaded?

(I don't feel strongly about this.)

Reply:
Lol, just refactored a little, and I'm doing exactly that. I agree, it makes more sense.